scripts/release/deploycdn.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+const upath = require( 'upath' );
+const { Listr } = require( 'listr2' );
+const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
+const parseArguments = require( './utils/parsearguments' );
+const { CKEDITOR5_ROOT_PATH, CDN_S3_BUCKET, S3_COPY_ARGS } = require( './utils/constants' );
+const cliArguments = parseArguments( process.argv.slice( 2 ) );
+
+const { version: packageJsonVersion } = require( upath.join( CKEDITOR5_ROOT_PATH, './package.json' ) );
+const version = cliArguments.nightly ? 'nightly' : packageJsonVersion;
+
+const tasks = new Listr( [
+	{
+		title: 'Upload files to CDN',
+		task: () => {
+			return tools.shExec(
+				`aws s3 cp ./release_cdn/ s3://${ CDN_S3_BUCKET }/ckeditor5/${ version }/ ${ S3_COPY_ARGS }`,
```

Comment:
Does it require any credentials? How it verifies if someone is authorized to perform this operation?